model/oauth/client.go

```diff
@@ -194,6 +194,27 @@ func GetNotifiables(i *instance.Instance) ([]*Client, error) {
 	return clients, nil
 }
 
+func GetConnectedUserClients(i *instance.Instance, limit int, bookmark string) ([]*Client, string, error) {
+	// Return clients with client_kind mobile, browser and desktop
+	var clients []*Client
+	req := &couchdb.FindRequest{
+		UseIndex: "connected-user-clients",
+		Selector: mango.And(mango.Gt("client_kind", ""), mango.Gt("client_name", "")),
```

Comment:
I think it is still WIP

Reply:
that's the intended query if that's what you're referring to.